tests/devapps/WAM/NetCoreWinFormsWam/Form1.cs

```diff
@@ -120,14 +120,14 @@ private IPublicClientApplication CreatePca(AuthMethod? authMethod)
                     builder = ToggleOldBroker(builder, true);
                     break;
                 case AuthMethod.WAMRuntime:
-                    builder = builder.WithBrokerPreview();
+                    builder = builder.WithExperimentalFeatures().WithBrokerPreview();
                     break;
                 case AuthMethod.SystemBrowser:
-                    builder = builder.WithBrokerPreview(false);
+                    builder = builder.WithExperimentalFeatures().WithBrokerPreview(false);
                     builder = ToggleOldBroker(builder, false);
                     break;
                 case AuthMethod.EmbeddedBrowser:
-                    builder = builder.WithBrokerPreview(false);
+                    builder = builder.WithExperimentalFeatures().WithBrokerPreview(false);
```

Comment:
It is not needed to add `WithExperimentalFeatures`

Reply:
This is needed for POP because it is still experimental.